packages/core/src/__tests__/create-ceramic.ts

```diff
@@ -0,0 +1,23 @@
+import Ceramic, { CeramicConfig } from '../ceramic';
+import { IpfsApi } from '@ceramicnetwork/common';
+import * as uint8arrays from 'uint8arrays';
+import * as sha256 from '@stablelib/sha256';
+import tmp from 'tmp-promise';
+import { Ed25519Provider } from 'key-did-provider-ed25519';
+
+export async function createCeramic(ipfs: IpfsApi, config?: CeramicConfig & { seed?: string }): Promise<Ceramic> {
+  const appliedConfig = {
+    stateStoreDirectory: await tmp.tmpName(),
```

Comment:
I think this should be `tmp.dir`?

Reply:
`tmpName` only generates a temporary name. Ceramic then in `LevelStateStore` makes a directory. IMO, `tmpName` is more aligned with real-world usage, when a user passes some name (may be non-existent yet), and the code creates whatever is needed at that path.

This is driven by my big disappointment with some CLI utilities that require you to create a folder/file with _correct_ (god knows what it means for every different command) permissions and structure before the first run. One tries to use the command for the first time, has no knowledge yet on how the command works, and then it spits out to you a (usually) cryptic message like "Incorrect path". Go figure what it meant, and what the command expects as an input.

Reply:
Hmm... so if `stateStoreDirectory` is just a string like "foo" and not a path, then where will the directory be created [here](https://github.com/ceramicnetwork/js-ceramic/blob/56912f56b74f0b97fdb95df972d6182389b2a7e4/packages/core/src/store/level-state-store.ts#L29-L30) (`storeRoot` will be the `stateStoreDirectory`)?  Won't it be a relative path to the CWD?  Or will it be from the root dir (ie `/foo`)?  Either way feels wrong to me?  I assume `tmp.dir` generates something in `/tmp`, which seems like the right place